test/functional/test_runner.py

```diff
@@ -130,6 +130,7 @@
     'feature_logging.py',
     'node_network_limited.py',
     'conf_args.py',
+    'wallet-fallbackfee.py',
```

Comment:
Could move up by a few lines to avoid merge conflicts

Reply:
Fixed.